app/controllers/api/clumps_controller.rb

```diff
@@ -0,0 +1,33 @@
+module API
+  class InvalidLocale < Exception;  end
+
+  class ClumpsController < ApplicationController
```

Comment:
Tbh it wasn't a conscious decision not to inherit from `APIController` - I used `CategoryContentsController` as a basis for this, and that inherits from `ApplicationController`.

Looking into it a bit further, looks like the reason for this is because `APIController` contains some functionality for looking up the site, whereas categories (and now clumps) aren't site specific, so don't make use of this behaviour.

That said though, this new controller does now introduce some duplication and I'm starting to wonder now if there's a better way to organise things (thinking maybe using concerns), so I'll have a look and see if I can come up with something.